src/arithmetic/algebraic_expression/algebraic_expression_construction.c

```diff
@@ -334,7 +334,7 @@ static AlgebraicExpression *_AlgebraicExpression_OperandFromEdge
 	AlgebraicExpression  *add               =  NULL;
 	AlgebraicExpression  *root              =  NULL;
 	AlgebraicExpression  *src_filter        =  NULL;
-	bool                 var_len_traversal  =  QGEdge_VariableLength(e);
+	bool                 var_len_traversal  =  QGEdge_RangeLength(e);
```

Comment:
Given the comment on qg_edge.h I believe we can revert this change back to its original version.